### ISSUE TITLE
coordinated modal ellipsis with wh argument

### ISSUE THREAD

**Issue by nschneid** [amrisi/amr-guidelines#240]
@mdtux89 raises this interesting example: "Who can afford it and who can't?"

Option A:

```lisp
(a / and
      :op1 (p / possible-01
            :ARG1 (a2 / afford-01
                  :ARG0 (a3 / amr-unknown)
                  :ARG1 (i2 / it)))
      :op2 (p2 / possible-01 :polarity -
            :ARG1 a2))
```

Option B:

```lisp
(a / and
      :op1 (p / possible-01
            :ARG1 (a2 / afford-01
                  :ARG0 (a3 / amr-unknown)
                  :ARG1 (i2 / it)))
      :op2 (p2 / possible-01 :polarity -
            :ARG1 (a4 / afford-01
                  :ARG0 (a5 / amr-unknown)
                  :ARG1 i2)))
```

The rationale for Option B is that the two "who"s refer to distinct sets of people, the haves and the have-nots, so perhaps this should be explicit with separate `amr-unknown`s.

A similar rationale would apply if the modals are removed, e.g. "Who likes fish and who doesn't?". Except then it is clearer that the main event ("likes") needs to be duplicated in order to host the negation in the second instance.

**Comment by mdtux89** [comment]
There is an instance of this in the LDC corpus, where they seem to follow Option B:

"Not knowing how much profit is in there, not caring who can afford it and who can't, these do not seem to be the principles of socialism."

(s / seem-01
      :ARG1 (p4 / principle :polarity -
            :poss (s2 / socialism)
            :domain (a / and
                  :op1 (k / know-01 :polarity -
                        :ARG1 (t2 / thing
                              :quant-of (p / profit
                                    :location (t / there))))
                  :op2 (c / care-01 :polarity -
                        :ARG1 (a3 / and
                              :op1 (p5 / person
                                    :ARG0-of (a4 / afford-01
                                          :ARG1 (i2 / it)
                                          :ARG1-of (p2 / possible-01)))
                              :op2 (p6 / person
                                    :ARG0-of (a6 / afford-01
                                          :ARG1 i2
                                          :ARG1-of (p3 / possible-01 :polarity -))))))))

I just wasn't sure if this was following some guidelines or an individual annotator decision.

**Comment by luciaelizabeth** [comment]
Is something like what is below possible? The “affording” refers to the same hypothetical event, but the participant is different.

Lucia

Option C:

 (a / and

      :op1 (p / possible-01

            :ARG1 (a2 / afford-01

                  :ARG0 (a3 / amr-unknown)

                  :ARG1 (i2 / it)))

      :op2 (p2 / possible-01 :polarity -

            :ARG1 a2

                  :ARG0 (a5 / amr-unknown)

                  :ARG1 i2)))

**Comment by nschneid** [comment]
> Option C: 
> ```
> (a / and
>        :op1 (p / possible-01
>              :ARG1 (a2 / afford-01
>                    :ARG0 (a3 / amr-unknown)
>                    :ARG1 (i2 / it)))
>        :op2 (p2 / possible-01 :polarity -
>              :ARG1 a2
>                    :ARG0 (a5 / amr-unknown)
>                    :ARG1 i2)))
> ```

No, since each variable refers to a unique node in the graph, in Penman notation it doesn't make sense to repeat a variable with different things underneath.

**Comment by luciaelizabeth** [comment]
Ah, correct. I don't think AMR uses the able-01 frame from Propbank (
http://verbs.colorado.edu/propbank/framesets-english-aliases/able.html),
but this sentence seems to read more like the following to me than the
other options:

(a / and
       :op1 (a / able-01
             :ARG1 (a2 / amr-unknown
             :ARG2 (a3 / afford-01
                   :ARG1 (i / it)))
       :op2 (a4 / able-01 :polarity -
             :ARG1 (a5 / air-unknown)
             :ARG2 a3))


Lucia Donatelli
http://luciadonatelli.georgetown.domains/

**Comment by nschneid** [comment]
> I don't think AMR uses the able-01 frame from Propbank

Correct. And if it did, the ARG1 of able-01 would probably be marked as coreferent with the ARG0 of afford-01 (as a case of control), so we'd face the same problem of whether to duplicate afford-01.
